File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/Price.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Price {

    @SerializedName("price")
    @Expose
    private String price;

    @SerializedName("price_raw")
    @Expose
    private Float priceRaw;

    public String getPrice() {
        return price;
    }

    public Float getPriceRaw() {
        return priceRaw;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/checkout/Price.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.checkout

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Price {

    @SerializedName("price")
    @Expose
    val price: String? = null

    @SerializedName("price_raw")
    @Expose
    val priceRaw: Float? = null

}
